### PR TITLE
build(common): specify explicit locales dir in package.json

### PR DESCRIPTION
**Pull request by filipesilva**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] angular.io application / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?
Locale file directory is implicit in `@angular/common`.

## What is the new behavior?
Locale file directory is explicit in `@angular/common`.

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

Followup from #19474.

### CODE REVIEW

File: packages/common/test/i18n/locale_data_folder.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {existsSync, readFileSync} from 'fs';
+import {basename, resolve} from 'path';
+
+export function main() {
+  describe('locale data folder', () => {
```
Comment (vicb) [inline packages/common/test/i18n/locale_data_folder.ts]: Don't we filter test files with `*_spec.ts` ? (ie is this spec executed at all ?)

**Review by vicb — CHANGES_REQUESTED** [review]
Please rename the file to be consistent and make sure the test is executed.

**Comment by vicb** [comment]
Sound like a `build` commit to me more than a `fix` - as this is not fixing anything. 
As such is there a real interest in merging in in the patch branch (given tools should support earlier version anyway) ?

File: packages/common/test/i18n/locale_data_folder.ts (hunk 1 above)
Comment (filipesilva) [reply packages/common/test/i18n/locale_data_folder.ts]: Right you are, my bad. Fixing.

**Comment by filipesilva** [comment]
@vicb I don't think it is important to merge to patch, no. In a minor would be fine.

I've changed the commit message and renamed the spec file, but I'm not sure what's going on with the pullapprove check.

**Comment by filipesilva** [comment]
@vicb I'm doing a unit test right now but that's the wrong approach. This sort of stuff is more of an integration test. The only package that seems to have integration tests right now is `compiler-cli` though.

Should I make a new script for `common` package integration tests? Or is there another approach you would rather me follow.

**Comment by filipesilva** [comment]
@ocombe also mentioned `./integration/i18n/` as a good place to add a test (via a node script).

**Review by vicb — APPROVED** [review]
Thanks for the updates

**Comment by filipesilva** [comment]
I followed @ocombe's suggestion to add an integration test instead, @vicb PTAL.

**Comment by ocombe** [comment]
Sounds good to me

**Review by vicb — APPROVED** [review]

**Review by jasonaden — APPROVED** [review]

**Comment by vicb** [comment]
@filipesilva remember to add the "merge" label for PRs to be merged !

**Comment by mhevery** [comment]
http://test/OCL:183314180:BASE:183314216:1516928283330:35ca6a6f

**Comment by filipesilva** [comment]
@vicb I cannot add labels in angular/angular :/

**Pull request closed without merging**